YoutubeExplode.Tests/PlaylistSpecs.cs

```diff
@@ -23,6 +23,11 @@ public async Task I_can_get_metadata_of_a_YouTube_playlist()
             playlist.Title.Should().Be("osu! Highlights");
             playlist.Author.Should().Be("Tyrrrz");
             playlist.Description.Should().Be("My best osu! plays");
+            playlist.Thumbnails.LowResUrl.Should().NotBeNullOrWhiteSpace();
+            playlist.Thumbnails.MediumResUrl.Should().NotBeNullOrWhiteSpace();
+            playlist.Thumbnails.HighResUrl.Should().NotBeNullOrWhiteSpace();
+            playlist.Thumbnails.StandardResUrl.Should().NotBeNullOrWhiteSpace();
+            playlist.Thumbnails.MaxResUrl.Should().NotBeNullOrWhiteSpace();
```

Comment:
`Thumbnails?.LowResUrl` to avoid null reference exception in case of a bug.
Same with others.

Reply:
Done